case-lib/pipeline.sh

```diff
@@ -8,7 +8,7 @@ func_pipeline_export()
     fi
     # got tplg_file, verify file exist
     tplg_path=$(func_lib_get_tplg_path "$1") || {
-        die "No available topology for pipeline export"
+        die "No topology found! Please configure TPLG env to point to you topology path."
```

Comment:
Many tests seem to have a `-t` option so setting TPLG is not the only possible solution.

It looks like this new error message will ask me set TPLG even when I make a typo in TPLG  that's confusing. It will also not show my typo.

How about:
```shell
die "Topology $1 not found, check the TPLG environment variable" 
```

Reply:
@marc-hb The default value for -t option is $TPLG. if No TPLG is set, and -t not specified, the "$1" will be empty.  How about this: `Topology not found, check TPLG environment variable or specify topology path with -t`

Reply:
- if `$1` is empty: `Topology not found, check TPLG`
- if `$1` is wrong: `Topology wrong_file not found, check TPLG`.

I don't mind mentioning `-t` in the message but it should also print `$1`.

Reply:
I'd like the -t in the message, updated